cursive-core/Cargo.toml

```diff
@@ -64,13 +65,40 @@ default-features = false
 optional = true
 version = "0.9"
 
+[dependencies.js-sys]
+version = "0.3.64"
+optional = true
+
+[dependencies.getrandom]
+optional = true
+version = "0.2"
+features = ["js"] 
```

Comment:
Am I missing something or is this dependency never used anywhere? Possible development-leftover?